tests/test_blocking.py

```diff
@@ -0,0 +1,225 @@
+import itertools
+import operator
+import random
+import unittest
+
+from anonlink import blocking
+
+
```

Comment:
Is it worth parameterising these tests by `datasets` and `samples` to allow easier checking of edge cases?

Reply:
Definitely! I don’t want to do it now (I’d like to start integrating with the entity service on Monday and I have a bunch more work to do before then) but I’ll do it before merging api-new to master.